File: 707_Tests/PFOv/PFOv/MainStarter.cs
using Hobis.Helpers;
using System;
using System.Runtime.InteropServices;
using System.Threading;
using System.Windows.Forms;


namespace PFOv
{
    public static class MainStarter
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThreadAttribute]
        public static void Main()
        {
            if (SingleInstanceCheck.IsTest())
            {
                Application.EnableVisualStyles();
                Application.SetCompatibleTextRenderingDefault(false);
                Application.Run(new FrmRoot());
            }
        }
    }
}

File: 707_Tests/CaseByStudy_001/CaseByStudy_001/MainStarter.vb
Imports System
Imports System.Windows.Forms

Public Module MainStarter
    Public Sub Main()
        Application.EnableVisualStyles()
        Application.SetCompatibleTextRenderingDefault(False)
        Dim t_mw As New MainWin()
        Application.Run(t_mw)
    End Sub
End Module
